package.json

```diff
@@ -84,6 +84,7 @@
     "classnames": "^2.2.6",
     "fuzzysort": "^1.1.4",
     "highcharts": "7",
+    "mathjs": "^6.1.0",
```

Comment:
Reason why new dependency is needed - YNAB's own calculator is residing in functions that are only available from their scope. I know of no OOPish interface or non-extremely hacky way how to plug it through to Toolkit.

"math.js" is stable and supported library that allows evaluation of arbitrary math expressions